src/utils/store.js

```diff
@@ -0,0 +1,19 @@
+import { applyMiddleware, createStore } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import thunk from 'redux-thunk';
+import { createLogger } from 'redux-logger';
+import reducers from '../reducers/index';
+
+const middleWare = [];
+middleWare.push(thunk);
+
+const loggerMiddleware = createLogger({
+  predicate: () => process.env.NODE_ENV === 'development',
+});
+
+middleWare.push(loggerMiddleware);
```

Comment:
Instead of pushing can we have `middleware([thunk, loggerMiddleware])`

Reply:
I can't test this now because I need to dispatch an action to test this but the suggestion looks nice to me. I will implement it and hopefully it works as expected

Reply:
Okay. That's fine.